### PR TITLE
Update layout atlas panel description

### LINKED ISSUE #1956: Composer: a image file format selector is added to atlas tab
See https://github.com/qgis/QGIS/pull/4749 for more information

### PR DESCRIPTION
**Pull request by DelazJ**
Fix #1956 
Build will fail because of reference in #2521

### CODE REVIEW

File: source/docs/user_manual/print_composer/create_output.rst [hunk 1]
```diff
@@ -156,88 +156,98 @@ use in |selectString| :guilabel:`Reference map`.
...
-books in an automated way. The concept is to use a coverage layer, which contains
-geometries and fields. For each geometry in the coverage layer, a new output will
-be generated where the content of some canvas maps will be moved to highlight the
-current geometry. Fields associated with this geometry can be used within text
-labels.
-
-Every page will be generated with each feature. To enable the generation
-of an atlas and access generation parameters, refer to the `Atlas generation`
-panel.This panel contains the following widgets (see figure_layout_atlas_):
+The print layout includes atlas functions that allow you to create map
```
Comment (anitagraser) [inline source/docs/user_manual/print_composer/create_output.rst]: Slightly restructured suggestion, including introduction to the term "atlas feature":

Atlas functions allow you to create map books in an automated way. Atlas uses the features of a vector or table layer called **coverage layer** and creates an output for each **atlas feature**. The most common usage is to zoom a map item to the current atlas feature. Further use cases include:

**Comment by DelazJ** [comment]
@anitagraser reformulation done. Anything else?

**Comment by anitagraser** [comment]
The rest looked good to me

**Comment by DelazJ** [comment]
Thanks. 
Back to 400 remaining issues :smile: :+1:

**Pull request merged**